<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalSetupWidget</class>
 <widget class="QWidget" name="SignalSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="Frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>184</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="typeComboBox">
      <item>
       <property name="text">
        <string>Raised Cosine Pulse</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gaussian Sine Pulse</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="amplitudeLabel">
        <property name="text">
         <string>Amplitude</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="amplitudeLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="frequencyLabel">
        <property name="text">
         <string>Frequency (MHz)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="frequencyLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="cyclesLabel">
        <property name="text">
         <string># Cycles</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="cyclesDoubleSpinBox">
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="previewPushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>Preview </string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/previewImage.png</normaloff>:/previewImage.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
